test-requirements.txt

```diff
@@ -0,0 +1,2 @@
+# odoo-addon-<module_name_with_dashes> @ git+https://github.com/OCA/<repository>/refs/pull/<PR_number>/head#subdirectory=setup/<module_name>
+odoo-addon-crm_claim @ git+https://github.com/OCA/crm/refs/pull/468/head#subdirectory=setup/crm_claim
```

Comment:
```suggestion
odoo-addon-crm_claim @ git+https://github.com/OCA/crm/pull/468/head#subdirectory=setup/crm_claim
```

Reply:
the solution was:
odoo-addon-crm_claim @ git+https://github.com/OCA/crm.git@refs/pull/468/head#subdirectory=setup/crm_claim